platform-coop-toolkit/coop-map-directory-index
# refactor: re: #102; creates & populates mdi_source

* [ x] I've read the [guidelines for Contributing to the Platform Co-op Toolkit](https://github.com/platform-coop-toolkit/.github/blob/master/CONTRIBUTING.md)
* [ x] This isn't a duplicate of an existing pull request

## Description

1. Creates `mdi.Source`
2. Populates `mdi.Source` from `accounts.Source`
3. Refactors code accordingly
4. Deletes `accounts.Source`

## Steps to test

1. Run migrations
2. Fire up `/admin`
3. Verify sources of known Organizations: 7 Stakes: ioo.coop, Adams Telephone: 2020 Ecosystem
4. Verify sources of known Individuals: nzimmerman@ocadu.ca: IDRC, 3rt@3rivers.net: 2020 Ecosystem

**Expected behavior:** 

Values from `accounts` correctly populated into `mdi`.

## Related issues

#102, #84
